src/index.html

```diff
@@ -1,13 +1,328 @@
+<!-- #region head -->
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="icon" href="images/logo.png">
+    <title>Nothing</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Space+Mono&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!-- #endregion -->
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <a href="#" class="top-bar__logo-link">
+              <img
+                src="images/logo.png"
+                alt="Logo Nothing"
+                class="top-bar__logo"
+              >
+            </a>
+            <div class="top-bar__icons">
+              <div class="container-for-phone">
+                <a href="tel:+1-234-5555-55-55" class="icon icon--phone"></a>
+                <a class="icon--number">+1 234 5555-55-55</a>
+              </div>
+              <a href="#menu" class="icon icon--menu"></a>
+            </div>
+          </div>
+          <div class="header__bottom">
+            <h1 class="header__title">
+              Bring joy back to the everyday
+            </h1>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <aside class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="top-bar menu__top">
+          <a href="#" class="top-bar__logo-link">
+            <img
+            src="images/logo.png"
+            alt="Logo Nothing"
+            class="top-bar__logo">
```

Comment:
Need to check and fix it everywhere
```suggestion
            <img
              src="images/logo.png"
              alt="Logo Nothing"
              class="top-bar__logo"
            >
```